Modules/EssUI/Sources/TableAdapterCore/TableAdapterHelper.swift

```diff
@@ -78,7 +78,7 @@ public class TableAdapterHelper {
         case .centeredButton:
             return 75.0
         case .attributedCenteredButton:
-            return 40.0
+            return 45.0
```

Comment:
Calculate button height due to component title text

Reply:
see, b802f32